x-pack/plugins/security/public/management/roles/edit_role/privileges/kibana/feature_table/sub_feature_form.tsx

```diff
@@ -33,14 +40,37 @@ export const SubFeatureForm = (props: Props) => {
     .getPrivilegeGroups()
     .filter((group) => group.privileges.length > 0);
 
+  const tooltip = useMemo(() => {
+    if (props.subFeature.privilegesTooltip) {
+      const tooltipContent = (
+        <EuiText>
+          <p>{props.subFeature.privilegesTooltip}</p>
+        </EuiText>
+      );
+      return (
+        <EuiIconTip
+          iconProps={{
+            className: 'eui-alignTop',
+          }}
+          type={'iInCircle'}
+          color={'subdued'}
+          content={tooltipContent}
+        />
+      );
+    }
+    return null;
```

Comment:
nit: Maybe move this return statement to the top of the block so that we return null when `!props.subFeature.privilegesTooltip`  is `true`, else we return the tooltip.